src/Nethermind/Nethermind.Monitoring/Metrics/MetricsRegistry.cs

```diff
@@ -40,9 +40,10 @@ public MetricsRegistry()
             RegisterMetrics(typeof(JsonRpc.Metrics));
             RegisterMetrics(typeof(InteropMetrics));
             RegisterMetrics(typeof(BeaconChain.Metrics));
+            RegisterMetrics(typeof(DataMarketplace.Consumers.Metrics));
         }
         
-        private void RegisterMetrics(Type type)
+        public void RegisterMetrics(Type type)
         {
```

Comment:
this has been changed in master recently to remove dependencies from Metrics to other projects